examples/torch/maml_ppo_half_cheetah_dir.py

```diff
@@ -9,6 +9,8 @@
 from garage.envs.base import GarageEnv
 from garage.experiment import LocalRunner
 from garage.experiment.deterministic import set_seed
+from garage.experiment.meta_evaluator import MetaEvaluator
```

Comment:
nit: which of these can these use canonical imports? `from garage.experiment import MetaEvaluator` etc.